encoding/dot/decode_test.go

```diff
@@ -0,0 +1,210 @@
+package dot_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"github.com/gonum/graph"
+	"github.com/gonum/graph/encoding/dot"
+	"github.com/gonum/graph/simple"
+	dotparser "github.com/graphism/dot"
```

Comment:
If this stays, the graphism imports should be in their own block between std and gonum.